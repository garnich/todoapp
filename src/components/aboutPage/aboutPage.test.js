import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import AboutPage from './aboutPage'

Enzyme.configure({ adapter: new Adapter() })

describe('Test AboutPage component', () => {
  it('<AboutPage>', () => {
    const mockedDate = new Date('06/25/2020')
    window.Date = jest.fn(() => mockedDate)

    const wrapper = shallow(<AboutPage />)

    expect(wrapper).toMatchSnapshot()
  })
})
