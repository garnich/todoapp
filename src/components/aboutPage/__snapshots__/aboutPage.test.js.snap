// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test AboutPage component <AboutPage> 1`] = `
<div
  className="col-12 about_page"
>
  <h1>
    About ToDo App
  </h1>
  <div
    className="about_page_content"
  >
    <div
      className="about_page_content_item"
    >
      <h2>
        ToDo App v.1
      </h2>
      <ul>
        For creating SPA used:
        <li>
          React
        </li>
        <li>
          Firebase
        </li>
        <li>
          WebPack
        </li>
        <li>
          Express
        </li>
        <li>
          Bootstrap
        </li>
        <li>
          Sass
        </li>
        <li>
          GitHub
        </li>
        <li>
          Herocu (as PaaS)
        </li>
      </ul>
    </div>
    <div
      className="about_page_content_item"
    >
      <h2>
        ToDo App v.1.1
      </h2>
      <p>
        1 days to go...
      </p>
      <ul>
        Updates of ToDo App:
        <li>
          Implement unit test
        </li>
      </ul>
    </div>
  </div>
</div>
`;
